### PR TITLE
Capstone Full Project With Features

### PR DESCRIPTION
**Pull request by Cosmosha**
The following features and requirement was implemented;

- Mobile version
- Mobile navigation bar with hamburger and close buttons
- Main program section
- Featured speakers section
- Dynamic HTML content with js
- Different screen displays.
- Project Documentation and Video

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,9 +91,9 @@
 - [Live Demo Link](https://cosmosha.github.io)
 
 
-## 🚀 Loom Intro <a name="live-demo"></a>
+## 🚀 Loom Intro Video <a name="live-demo"></a>
 
-- [Loom Intro]()
+- [Loom video link](https://www.loom.com/share/6124b8ad67674ea9bef2edccd550795b)
```
Comment (danielufeli) [inline README.md]: - [ ] Nice work you put into making your video presentation, has a requirement you need to add a highlight about a piece of code you find interesting or are proud of making.
- [ ] In your "Acknowledgements" section kindly add credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).
- [ ] Kindly fix your MIT Link it is going to a 404 error page

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @Cosmosha , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights
- Nice commits messages✔️
- Clean Code ✔️

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 2]
```diff
@@ -147,7 +147,7 @@ <h3 class="event-title">Partners</h3>
     <footer>
         <div class="footer-info">
             <img src="./images/ghtech.png" alt="logo">
-            <p>@2023 Ghana Tech Summit <br> All Right Reserved</p>
+            <p>@2023 Ghana Tech Summit <br> All Rights Reserved</p>
         </div>
     </footer>
```
Comment (mckent05) [inline index.html:152]: **OPTIONAL** - Some parts of your project do not look exactly like the design template, please fix this 🙏.Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏.
 Your project:
-- There should be five cards in the main program section, your project has just four. Please fix this 👍 👍 
nzdzdr
Design Template:
efwef
**OPTIONAL**- in your desktop version, for screen sizes `>768px`, the speakers section does not look exactly lie the design template, please fix this 🙏 
Your Project:
-- This sections should have six cards, aligned in two columns and three rows, please fix this 🙏 
ethth

Design template:
speakers behance

**Review by mckent05 — APPROVED** [review]
Hi @Cosmosha, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**